modules/fedimint-ln-tests/tests/tests.rs

```diff
@@ -903,6 +916,7 @@ mod fedimint_migration_tests {
     async fn snapshot_client_db_migrations() -> anyhow::Result<()> {
         snapshot_db_migrations(
             "lightning-client-v0",
+            Some(LEGACY_HARDCODED_INSTANCE_ID_LN),
```

Comment:
We should not be adding more places using `LEGACY_HARDCODED_`.

Reply:
Remove all instances of `LEGACY` from the db migration module tests